TeslaMateAgile.Tests/IntegrationTests.cs

```diff
@@ -78,4 +78,46 @@ public async Task IntegrationTests_Awattar()
         Assert.LessOrEqual(priceData.Min(x => x.ValidFrom), from);
         Assert.GreaterOrEqual(priceData.Max(x => x.ValidTo), to);
     }
+
+    //[Ignore(IntegrationTest)]
+    [Test]
+    public async Task IntegrationTests_Nordpool()
+    {
+        var configBuilder = new ConfigurationBuilder()
+            .AddInMemoryCollection(new Dictionary<string, string>
+            {
+                ["Nordpool:BaseUrl"] = "http://www.nordpoolspot.com/api/marketdata/page/10",
+                ["Nordpool:Currency"] = "DKK",
+                ["Nordpool:Region"] = "DK1"
+            });
+
+        var config = configBuilder.Build();
+
+        var services = new ServiceCollection();
+        services.AddHttpClient();
+        services.AddOptions<NordpoolOptions>()
+                        .Bind(config.GetSection("Nordpool"))
+                        .ValidateDataAnnotations();
+        services.AddHttpClient<IPriceDataService, NordpoolService>((serviceProvider, client) =>
+        {
+            var options = serviceProvider.GetRequiredService<IOptions<NordpoolOptions>>().Value;
+            var baseUrl = options.BaseUrl;
+            if (!baseUrl.EndsWith("/")) { baseUrl += "/"; }
+            client.BaseAddress = new Uri(baseUrl);
+        });
+        var priceDataService = services.BuildServiceProvider().GetRequiredService<IPriceDataService>();
+
+        var from = new DateTimeOffset(2022, 2, 20, 0, 0, 0, new TimeSpan(1, 0, 0));
+        var to = new DateTimeOffset(2022, 2, 20, 23, 59, 0, new TimeSpan(1, 0, 0));
+
+        var priceData = await priceDataService.GetPriceData(from, to);
+
+        foreach (var price in priceData)
+        {
+            Console.WriteLine($"{price.ValidFrom} - {price.ValidTo} - {price.Value}");
+        }
+
+        Assert.LessOrEqual(priceData.Min(x => x.ValidFrom), from);
+        Assert.GreaterOrEqual(priceData.Max(x => x.ValidTo), to);
```

Comment:
This integration test currently fails as we the price data does not span the full range of the input date ranges, looks like it could be related to output timezones:

```log
Message: 
    Expected: less than or equal to 2022-02-20 00:00:00+01:00
    But was:  2022-02-20 00:00:00+00:00

Standard Output: 
    20/02/22 00:00:00 +00:00 - 20/02/22 01:00:00 +00:00 - 0.00000
    20/02/22 01:00:00 +00:00 - 20/02/22 02:00:00 +00:00 - 0.00000
    20/02/22 02:00:00 +00:00 - 20/02/22 03:00:00 +00:00 - 0.00000
    20/02/22 03:00:00 +00:00 - 20/02/22 04:00:00 +00:00 - 0.00000
    20/02/22 04:00:00 +00:00 - 20/02/22 05:00:00 +00:00 - 0.00000
    20/02/22 05:00:00 +00:00 - 20/02/22 06:00:00 +00:00 - 0.00000
    20/02/22 06:00:00 +00:00 - 20/02/22 07:00:00 +00:00 - 0.00000
    20/02/22 07:00:00 +00:00 - 20/02/22 08:00:00 +00:00 - 0.00000
    20/02/22 08:00:00 +00:00 - 20/02/22 09:00:00 +00:00 - 0.00000
    20/02/22 09:00:00 +00:00 - 20/02/22 10:00:00 +00:00 - 0.00000
    20/02/22 10:00:00 +00:00 - 20/02/22 11:00:00 +00:00 - 0.00000
    20/02/22 11:00:00 +00:00 - 20/02/22 12:00:00 +00:00 - 0.00000
    20/02/22 12:00:00 +00:00 - 20/02/22 13:00:00 +00:00 - 0.00000
    20/02/22 13:00:00 +00:00 - 20/02/22 14:00:00 +00:00 - 0.00000
    20/02/22 14:00:00 +00:00 - 20/02/22 15:00:00 +00:00 - 0.00000
    20/02/22 15:00:00 +00:00 - 20/02/22 16:00:00 +00:00 - 0.000
    20/02/22 16:00:00 +00:00 - 20/02/22 17:00:00 +00:00 - 0.00000
    20/02/22 17:00:00 +00:00 - 20/02/22 18:00:00 +00:00 - 0.00000
    20/02/22 18:00:00 +00:00 - 20/02/22 19:00:00 +00:00 - 0.00000
    20/02/22 19:00:00 +00:00 - 20/02/22 20:00:00 +00:00 - 0.00000
    20/02/22 20:00:00 +00:00 - 20/02/22 21:00:00 +00:00 - 0.00000
    20/02/22 21:00:00 +00:00 - 20/02/22 22:00:00 +00:00 - 0.00000
    20/02/22 22:00:00 +00:00 - 20/02/22 23:00:00 +00:00 - 0.00000
```